examples/svelte/src/App.svelte

```diff
@@ -0,0 +1,66 @@
+<script>
+    import { router } from './router.js';
+    import { uuid } from './utils.js';
+
+    import Header from './Header.svelte';
+    import Footer from './Footer.svelte';
+    import Item from './Item.svelte';
+
+    import "./app.css";
+    import "todomvc-app-css/index.css";
+    import "todomvc-common/base.css";
```

Comment:
I had previously left these out so they didn't affect the bundle size of the app itself (since some people are concerned with that), but if this is commonly bundled with the other implementations then 👍

Reply:
Yeah - other apps typically import the base.css as well.